gnosis-imap/gnosis-imap-spec.ini

```diff
@@ -6,27 +6,17 @@
 [root]
 code: {MASTER_COPY_CODE}
 comment:
-k: (#execute => #halt) ~> _
-callStack: _
-wordStack: .WordStack => _
-localMem: .Map => _
-pc: 0 => _
-gas: #infinity => _
-memoryUsed: 0 => _
 callDepth: CD
-log: _ => _
-refund: _ => _
-coinbase: _ => _
```

Comment:
why did you remove this? you can just overwrite by specifying them in your spec.

Reply:
maybe you think that this helps the readability and less error-prone?